src/SFA.DAS.AssessorService.Application.Api/Controllers/OrganisationStandardController.cs

```diff
@@ -156,5 +155,20 @@ public async Task<IActionResult> AddOrganisationStandard([FromBody] Organisation
                 return BadRequest(new EpaoStandardVersionResponse(ex.Message));
             }
         }
+
+        [HttpPost("organisationstandard/withdraw", Name = "WithdrawStandard")]
+        [ValidateBadRequest]
+        [SwaggerResponse((int)HttpStatusCode.NoContent)]
+        [SwaggerResponse((int)HttpStatusCode.BadRequest, Type = typeof(IDictionary<string, string>))]
+        [SwaggerResponse((int)HttpStatusCode.InternalServerError, Type = typeof(ApiResponse))]
+        public async Task<IActionResult> WithdrawStandard(
+           [FromBody] WithdrawStandardRequest request)
+        {
+            _logger.LogInformation($"Received Withdraw Standard Request from {request.EndPointAssessorOrganisationId} for {request.StandardCode} on {request.WithdrawalDate}");
+
+            await _mediator.Send(request);
+
+            return NoContent();
+        }
```

Comment:
SonarCloud says that the unsanitised `request` is vulnerable to log injection attacks. This is an inner API, but I don't know whether the outer API accounts for this. I think it's worth considering the warning:
https://sonarcloud.io/project/issues?resolved=false&types=VULNERABILITY&pullRequest=1557&id=SkillsFundingAgency_das-assessor-service&open=AYp1bAB9yC-mESBRCjZ-&tab=code

Reply:
I have resolved that as Won't fix as we do not share the logs with other applications